src/lib/zod/register-schema.ts

```diff
@@ -31,7 +30,7 @@ export const RegisterFormSchema = {
       .min(5, { message: 'username must be at least 5 characters long.' })
       .max(10, { message: 'username must be at most 10 characters long.' })
       .refine(value => value !== '', { message: 'This field is required.' }),
-      // .regex(/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]+$/, { message: 'username must include aplphabet and numbers' }): 추후 가능성을 위해 주석처리
+    // .regex(/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]+$/, { message: 'username must include aplphabet and numbers' }): 추후 가능성을 위해 주석처리
```

Comment:
별건 아니긴 한데 이 주석에도 오타가 있긴 하네요

```suggestion
    // .regex(/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]+$/, { message: 'username must include alphabet and numbers' }): 추후 가능성을 위해 주석처리
```